css/custom.scss

```diff
@@ -221,6 +225,45 @@ $link-active: $link-hover;
   }
 }
 
+.section {
```

Comment:
Where is the section used?

Reply:
The `class` section contains all the _posts_ cards. He used CSS to move the section downwards (_2rem_ down from the top). Doesn't make that much difference, in my opinion. If you agree, I'd like to remove it, please.

Reply:
Remove then :smile: